pkgs/tools/audio/beets/default.nix

```diff
@@ -215,13 +232,7 @@ in pythonPackages.buildPythonApplication rec {
       s,"mp3val","${mp3val}/bin/mp3val",
     }' beetsplug/badfiles.py
   '' + optionalString enableConvert ''
-    sed -i -e 's,\(util\.command_output(\)\([^)]\+\)),\1[b"${ffmpeg_3.bin}/bin/ffmpeg" if args[0] == b"ffmpeg" else args[0]] + \2[1:]),' beetsplug/convert.py
-  '' + optionalString enableReplaygain ''
-    sed -i -re '
-      s!^( *cmd *= *b?['\'''"])(bs1770gain['\'''"])!\1${bs1770gain}/bin/\2!
```

Comment:
./replaygain-default-ffmpeg.patch doesn't include the change done here, equivalent for ffmpeg, which was present in a previous version of the PR.

Reply:
Okay, I do not want to bother with fixing this now. I will revert to the sed behavior and it can be solved in a subsequent PR.